pkg/distributor/otel.go

```diff
@@ -484,10 +498,13 @@ func otelMetricsToMetadata(addSuffixes bool, md pmetric.Metrics) []*mimirpb.Metr
 func otelMetricsToTimeseries(
 	ctx context.Context,
 	converter *otlpMimirConverter,
-	addSuffixes, enableCTZeroIngestion, enableStartTimeQuietZero bool,
+	addSuffixes,
+	enableCTZeroIngestion,
+	enableStartTimeQuietZero,
+	keepIdentifyingResourceAttributes,
+	convertHistogramsToNHCB,
+	promoteScopeMetadata bool,
```

Comment:
Nit: I would consider passing this as a struct (unexported, just defined in this package) because there's a high chance of these bools accidentally changing order or shifting and not being properly covered by a test.

Reply:
Or maybe just pass the `limits` to that function.

Reply:
Will take a look tomorrow probably :) Thanks!